packages/router-api/src/msg.rs

```diff
@@ -25,12 +25,23 @@ pub enum ExecuteMsg {
      * Router Admin Methods
      * All the below messages should only be called by the router admin
      */
-    // Freezes a chain, in the specified direction.
+    /// Freezes a chain, in the specified direction.
     FreezeChain {
         chain: ChainName,
         direction: GatewayDirection,
     },
-    // Unfreezes a chain, in the specified direction.
+    /// If no filter is provided, freezes all chain connections in every direction. Should only be called in emergencies.
+    /// If the filter is set, only the specified chains will be frozen.
+    FreezeAllChains { chains: Option<Vec<ChainName>> },
+
+    /// Unfreezes the specified chains in the specified directions. In contrast to FreezeAllChains,
+    /// this does not allow unfreezing all chains, because that command is too dangerous.
+    /// It would make it too easy to accidentally unfreeze a chain that should not be unfrozen.
+    UnfreezeChains {
+        chains: Vec<(ChainName, GatewayDirection)>,
```

Comment:
```suggestion
        chains: Vec<ChainName>,
```
Since this batch cmd is really only needed when unfreezing multiple chains from a freeze all status, I think the direction can be treated as bidirectional implicitly to keep the command simpler. UnfreezeChain can be used in other cases

Reply:
Based on discussion in Slack, we can have cases of unidirectional connections when deprecating a chain so allowing the direction to be configured when unfreezing makes sense then